draft-birkholz-rats-corim.md

```diff
@@ -109,15 +109,15 @@ See Body {{mybody}}.
 
 New indexes for Software Tag Link Relationship Values needs to be registered as below:
 
- +==========+========================+============================================+
- | Index    | Relationship Type Name | Description                                |
- +==========+========================+============================================+
- | 12       | needs                  | links to a module this software needs      |
- +----------+------------------------+--------------------------------------------+
- | 13       | runs-on                | links to a module this software runs       |
- +----------+------------------------+--------------------------------------------+
- | 14       | part-of                | links to a module this software is part of |
- +----------+------------------------+--------------------------------------------+
- | 16       | m-supplemental         | provides additional information if a module|
-  +----------+------------------------+-------------------------------------------+
+ +=======+========================+============================================+
+ | Index | Relationship Type Name | Description                                |
+ +=======+========================+============================================+
+ | 12    | needs                 | links to a module this software needs      |
+ +-------+-----------------------+--------------------------------------------+
+ | 13    | runs-on               | links to a module this software runs       |
+ +-------+-----------------------+--------------------------------------------+
+ | 14    | part-of               | links to a module this software is part of |
+ +-------+-----------------------+--------------------------------------------+
+ | 16    | m-supplemental        | provides additional information if a module|
+  +------+------------------------+-------------------------------------------+
```

Comment:
This should be a [markdown table](https://www.markdownguide.org/extended-syntax/#tables), so something like:

```
| Index | Relationship Type | Definition |
|---
| 13 | runs-on | links to a CoMID tag describing the module on which this software runs |
...
{: #tbl-new-coswid-to-comid-link-ownership-values title="Link Ownership Values"}
```